classes/services/queryBuilders/PKPUserQueryBuilder.inc.php

```diff
@@ -37,6 +37,12 @@ class PKPUserQueryBuilder extends BaseQueryBuilder implements EntityQueryBuilder
 	/** @var array list of role ids */
 	protected $roleIds = null;
 
+	/** @var int Assigned as editor to this category id */
+	protected $assignedToCategoryId = null;
+
+	/** @var int Assigned as editor to this section id */
+	protected $assignedToSectionId = null;
```

Comment:
This is fine to leave as-is. Just a note that I have begun to prefer using plural arguments that accept an array, like:

```
protected $asssignedToSectionIds = []
```

This just lets the querybuilders support a few more cases with very little overhead.

Reply:
I think this will be revisited soon anyway if we move to SubmissionGroups.